packages/metal-drag-drop/src/Drag.js

```diff
@@ -723,7 +723,9 @@ class Drag extends State {
 	 */
 	toElements_(elementOrSelector) {
 		if (core.isString(elementOrSelector)) {
-			let matched = this.container.querySelectorAll(`${elementOrSelector}:not([aria-grabbed="true"])`);
+			let matched = this.container.querySelectorAll(
```

Comment:
Hey @diegonvs, I don't fully understand this based on the commit message. Is it just source formatting, or is it meant to do something else?

Reply:
WTF, commit message is wrong. I'll fix it!

Reply:
just squash both commits into one and we're good to go 😉

Reply:
https://github.com/metal/metal-plugins/pull/101/commits
Now everything is okay :)